doc/source/data/getting-started.rst

```diff
@@ -4,113 +4,144 @@ Getting Started
 ===============
 
 A Ray :class:`Dataset <ray.data.Dataset>` is a distributed data collection. It holds
-a list of Ray object references pointing to distributed data *blocks*, and has APIs
-for distributed data loading and processing. Each block holds an ordered collection
-of items in either an `Arrow table <https://arrow.apache.org/docs/python/data.html#tables>`__
-(when creating from or transforming to tabular or tensor data) or a Python list (for non-tabular Python objects).
+references to distributed data *blocks*, and exposes APIs for loading and processing
+data.
 
-In this tutorial you will learn how to:
+Install Ray Data
+----------------
 
-- Create and save a Ray ``Dataset``.
-- Transform a ``Dataset``.
-- Pass a ``Dataset`` to Ray tasks/actors and access the data inside.
+To install Ray Data, run:
 
-.. tip::
+.. code-block:: console
 
-   Run ``pip install "ray[data]"`` to get started!
+    $ pip install 'ray[data]'
 
-Creating and Saving Datasets
-----------------------------
+To learn more about installing Ray and its libraries, read
+:ref:`Installing Ray <installation>`.
 
-You can create a Dataset from Python objects. These objects can be held inside
-Dataset as the plain Python objects (where the schema is a Python type), or as
-Arrow records (in which case their schema is Arrow).
+Create a dataset
```

Comment:
Shall we keep uppercase to be consistent with other places? `Create a Dataset`

Reply:
Made all of them lowercase for consistency.